src/core/plugins/spec/actions.js

```diff
@@ -100,7 +100,7 @@ export const resolveSpec = (json, url) => ({specActions, specSelectors, errActio
     url = specSelectors.url()
   }
 
-  let getLineNumberForPath = AST.getLineNumberForPath ? AST.getLineNumberForPath : () => undefined
```

Comment:
The changes here (and elsewhere in `actions.js`) cause fatal errors when a document with an invalid $ref is loaded, since `resolveSpec` and `debResolveSubtrees` always expect `getLineNumberForPath` to be a function.

Usage of the function:

https://github.com/geraldglynn/swagger-ui/blob/f6b50bf6248fedc0e0f25e704988f7136cb7d4ab/src/core/plugins/spec/actions.js#L123

https://github.com/geraldglynn/swagger-ui/blob/f6b50bf6248fedc0e0f25e704988f7136cb7d4ab/src/core/plugins/spec/actions.js#L192

Stack trace:

```
Uncaught (in promise) TypeError: Cannot read property 'getLineNumberForPath' of undefined
    at _callee2$ (574:437)
    at tryCatch (800:62)
    at Generator.invoke [as _invoke] (800:296)
    at Generator.prototype.(:3200/anonymous function) [as next] (webpack-internal:///800:114:21)
    at step (662:17)
    at eval (662:35)
    at new Promise (<anonymous>)
    at new F (VM63144 24:36)
    at eval (662:14)
    at invokeFunc (803:95)
```

Document you can use to test: https://gist.githubusercontent.com/shockey/4293e4be71095746221a21ca91de86eb/raw/baae0f50669b21f9bbfe7d363f5c55914451bdd9/simple-petstore-with-definition-ref.yaml